### DISCUSSION TITLE
Use `translation strings` for collection fields

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#12868] [Feature Requests]
## How it is
Right now collection fields can be renamed or translated with some custom translation collection or so:
![image](https://user-images.githubusercontent.com/10547444/164106054-7bbe5cf5-4ce3-4f71-8528-cd54b85a7cc7.png)

## How it could be
I think it would be better if we would maybe just have an input field called `custom name` or `translation` or something completely different.
So the idea would be that this field would be to `override` the current name that is presented to the user and optionally if you want it to be multi lingual, you can insert a translation string:

![image](https://user-images.githubusercontent.com/10547444/164106537-d730a78a-4fb2-4b4e-92c2-91f6b2d9784b.png)

Benefits:
- Much less clicks necessary
- Most of the times field names repeat themselves through multiple collections. For example i have a `title` or `subtitle` or `image` field in most if not every single collection.

**Comment by rijkvanzanten** [comment]
Fully agree with this! The new translations UI is much cleaner than the previous setup. We do have to write a custom migration to make sure we move the configured translations to the new format though. @azrikahar is this something you'd like to pick up? 🙂

**Reply by azrikahar** [reply]
Sounds good to me as well! Just to confirm the following details before proceeding:

## Fields

the "After" here should fit the requirement here right?

- Before

   ![chrome_UNjCwqFmcb](https://user-images.githubusercontent.com/42867097/164176097-96951cd4-ea43-458b-a9ce-9a2134991fc8.png)
   
- After   

   ![chrome_H114qQFYWD](https://user-images.githubusercontent.com/42867097/164176302-8e4de8ed-cd1d-4152-9246-2772a3550d3a.png)

## Collections

For collections, there are also Singular and Plural fields, but technically they are in the same column since they are json. If we were to make the same move here, would it mean we'll move the 3 fields directly on the collection page itself? If so, this would mean we'd need to make a new custom system interface that uses 3 input translated strings as a single field right? 🤔 Just want to make sure I'm envisioning this change requirement correctly.

https://user-images.githubusercontent.com/42867097/164176511-55f21038-d3b5-4fe9-b06b-29e43777f006.mp4

**Reply by LasseRosenow** [reply]
One Idea I have is to maybe only have one input field for collections as well and extending translation strings to have an optional plural field.

That could also be helpful for other use cases in the future maybe, if there might be other places that could be using them.

Also I would then mark those translations strings that have a plural value with some sort of badge or something so that we know which fields have plurals and which not when we select them

**Reply by benhaynes** [reply]
Ohh, this is a great idea...

**Reply by piotr-cz** [reply]
I'd start with the *Field Name Translations* since it's easier and once it's settled add translatable collection names.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by yonnic** [comment]
its end of 2023 and still no way to change the label of a field :(

**Reply by benhaynes** [reply]
There are field level translations to change the label... Could you explain why that feature isn't enough?

**Reply by br41nslug** [reply]
Check the message above, this is a closed feature request as it didnt get enough interest from the community. Feel free to create a feature request with the changes you want to see in labels instead of reacting to old, closed and possibly unrelated feature requests.

**Comment by yonnic** [comment]
> There are field level translations to change the label... Could you explain why that feature isn't enough?

thanks for ultra fast reply. The problem is with system fields, i would like to rename the user_created field to "Author" but the translation setting is missing here

Edit: guess need to create a new feature request for this

**Reply by br41nslug** [reply]
Not sure what that has to do with displays or the global "translation strings" but sounds like you're looking for the existing "field name translations" not a new feature 🤔 

![image](https://github.com/directus/directus/assets/9389634/5743284b-20ba-474b-87d0-395b34140d81)
![firefox_FPHiTDjp6L](https://github.com/directus/directus/assets/9389634/4a4b03fa-5bed-482d-9af5-20624e80a995)
![image](https://github.com/directus/directus/assets/9389634/52a0956b-3c07-4ff8-9a04-b82045f8c488)
